src/components/StackList/index.js

```diff
@@ -1,5 +1,6 @@
-import React, { Component } from 'react';
+import React, { Component, Fragment } from 'react';
```

Comment:
Ey! Great job! There is no need for the Frangment destructuring here though, just Component ;)